healthcheck/utils.py

```diff
@@ -0,0 +1,26 @@
+import errno
+import os
+
+
+def file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
```

Comment:
Yeah I didn't realise that was happening. Your comment below (https://github.com/yola/healthcheck/pull/21/files#r43479887) is spot on, that EACCESS return true is wrong. I thought this would return something different on the status endpoint, but I was wrong.